### ISSUE TITLE
Extra menu is not displayed

### ISSUE THREAD

**Issue by Marcy135** [matsn0w/MISS-ELS#110]
Hey, I don't get the extra menu when I press the set button. Only when I restart the MissELS, it works. But when I reconnect then I have the same problem. It always works only after a restart of the script when you are online.

FiveM build: 2545
MISS ELS version: 2.1.1

**Comment by matsn0w** [comment]
Hey, thank you for reporting this issue! This is a duplicate of #63, but I think this is starting to become an annoying problem that must be resolved. I'll keep this issue open to further investigate.

**Comment by matsn0w** [comment]
Hey @Marcy135, I think I've fixed your issue with #117. Can you please download the branch from that PR and test if it works?

**Comment by Marcy135** [comment]
> Hey @Marcy135, I think I've fixed your issue with #117. Can you please download the branch from that PR and test if it works?

I have tried it now, unfortunately the problem still exists. Also I get an error in the console. 
![Screenshot_9](https://user-images.githubusercontent.com/31376176/173552477-1b238421-6f35-4d7a-9e96-ac2fcb135252.png)

**Comment by matsn0w** [comment]
Hey, I've also noticed that. My mistake! It's already fixed in the latest `develop`.

Strange that it works on my servers. I'll investigate further.
